src/config/app.js

```diff
@@ -1,15 +1,42 @@
 // istanbul ignore file
+import Joi from 'joi';
 import { createRequire } from 'module';
 import { fileURLToPath } from 'url';
 import path from 'path';
 
+// Define config schema
+const schema = Joi.object({
```

Comment:
I think schemas should be in a separate module for seperation of concerns and reusability.

Reply:
Done